manifests/l/LuculentSystems/AltA2DP/1.2.0/LuculentSystems.AltA2DP.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=NVS0.CRLF.7-2-10.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: LuculentSystems.AltA2DP
+PackageVersion: 1.2.0
```

```suggestion
PackageVersion: 1.2.0.1
```
This must match Control Panel.

You also need to change `manifests/l/LuculentSystems/AltA2DP/1.2.0` to `manifests/l/LuculentSystems/AltA2DP/1.2.0.1`

---

Thank you for your suggestion, but we've been using 3-segment version number for all public releases.
And, correct me if I'm wrong, but there's no WinGet requirement about how to use the PackageVersion.

---

> And, correct me if I'm wrong, but there's no WinGet requirement about how to use the PackageVersion.

In https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.4.0/installer.md#fields for `PackageVersion` it says
- The current best practice is to use the value reported in Add / Remove Programs when this version of the package is installed.
- This practice may seem contrary to using semantic versioning, but it provides the best end to end experience for customers.

^ Add & Remove Programs (ARP) is Control Panel (Win32) / Installed apps (UWP).

It is possible to use `PackageVersion: 1.2.0` and use `DisplayVersion: 1.2.0.1` as part of AppsAndFeaturesEntries but I believe you will have to do that for all previous versions submitted to the WinGet package repository.

---

Please update file path to match this change.